frontend/src/pages/LandingPage.js

```diff
@@ -1,33 +1,36 @@
 // SPDX-License-Identifier: MIT
 // SPDX-FileCopyrightText: 2021 Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>
 
-import * as React from "react";
+import React from "react";
 
 import WelcomeBanner from "../components/Banner";
 import HeaderTypography from "../components/HeaderTypography";
 
 import Grid from "@mui/material/Grid";
 
 import AnnouncementPage from "./AnnouncementPage";
+import ExampleNftImg22 from "../assets/img/nftExamples/image_part_022.png";
+import ExampleNftImg21 from "../assets/img/nftExamples/image_part_021.png";
+import NftCardHorizontallyScrollableList from "../components/NftCardHorizontallyScrollableList";
+let dummyData = [
+	{ title: "Nürnberg NFT 021", price: "1.0", nftPageUrl: "/drop", imgUrl: ExampleNftImg21, description: "There is something cool about this text. When this text gets to long the text will be automatically cut off. My absolutely biggest secret is that I love cookies." },
+	{ title: "Nürnberg NFT 022", price: "0.0001", nftPageUrl: "/drop", imgUrl: ExampleNftImg21, description: "text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text " },
+	{ title: "Nürnberg NFT 023", price: "0.0002", nftPageUrl: "/drop", imgUrl: ExampleNftImg22, description: "This text can be descriptive." },
+];
 
 export default function LandingPage({ user }) {
-	const DropsListPlaceholder = () => {
-		const titleStyle = { fontSize: 20, fontWeight: 700, };
-
-		return <div style={{ backgroundColor: "orange", height: "100px" }}>
-			<HeaderTypography style={titleStyle}>Drops here!</HeaderTypography>
-		</div>;
-	};
 
 	return (
-		<Grid container direction="row" rowSpacing="50px" columnSpacing="50px">
+		<Grid container direction="row" rowSpacing="55px" columnSpacing="50px">
```

Comment:
Why is this 55px needed? In the mockups it rather is like 50 when subtracting the free space between border and text.

Reply:
When you measure it in Figma it is 55.

Reply:
Okay. I subtracted 5 because I felt there is more padding in the text, but it won't matter as long as the announcement sidebar is aligned properly.